  frame: /map
  map_resolution: 0.05
  map_origin_x: -20.025
  map_origin_y: -5.025
  map_origin_z: 0.0
  map_size_x: 600
  map_size_y: 300
  map_size_z: 100
  base_height_disc: 5
  unnocupied_prob_threshold: 0.3
  occupancy_prob_threshold: 0.8
  lidar_update_increment: 0.9
  lidar_update_decrement: 0.05
  
  
  horizontal_lidar_pose_tf_name: /base_laser_link
  horizontal_lidar_topic_name: /base_scan
  vertical_lidar_pose_tf_name: /tilt_laser_link
  vertical_lidar_topic_name: /tilt_scan
  camera_pose_tf_name: /base_laser_link
  camera_scan_topic_name: /camera_scan_node/camera_scan
  
  map_publish_rate: 6.0
  number_of_scans_to_skip: 1
  publish_debug_messages: false
